st2common/st2common/hooks.py

```diff
@@ -30,6 +31,18 @@
 
 LOG = logging.getLogger(__name__)
 
+# A list of method names for which we don't want to log the result / response
+RESPONSE_LOGGING_METHOD_NAME_BLACKLIST = [
+    'get_all'
+]
+
+# A list of controller classes for which we don't want to log the result / response
+RESPONSE_LOGGING_CONTROLLER_NAME_BLACKLIST = [
```

Comment:
To clarify it - he just moved this code around. This code already existed before. I added it during the logging changes.

We now log HTTP response for all the endpoints except "list" and those 3 blacklisted endpoints. Those endpoints are blacklisted because they could potentially contain a massive response and we don't want to log that.